molgenis-metadata-manager/src/main/frontend/src/components/MetadataManagerEntityEditForm.vue

```diff
@@ -123,7 +123,13 @@
     name: 'metadata-manager-entity-edit-form',
     methods: {
       saveEntityType () {
-        this.$store.dispatch(SAVE_EDITOR_ENTITY_TYPE)
+        if (this.editorEntityType.idAttribute === null || this.editorEntityType.idAttribute === undefined) {
+          this.$store.commit(CREATE_ALERT, {type: 'warning', message: 'ID attribute can not be empty'})
+        } else if (this.editorEntityType.labelAttribute === null || this.editorEntityType.labelAttribute === undefined) {
+          this.$store.commit(CREATE_ALERT, {type: 'warning', message: 'Label attribute can not be empty'})
```

Comment:
This is good, but does not stop the entity being created on the server side. The code creating the entity should check the modal validity. The checking model validity on the client side could maybe be grouped into a dedicated named function.